Develop an SQL schema for SOPs

```diff
@@ -0,0 +1,29 @@
+-- Table: public.SOPs
+
+-- DROP TABLE public."SOPs";
+
+CREATE TABLE public."SOPs"
+(
+    id text COLLATE pg_catalog."default" NOT NULL,
```

Comment:
The `id` should be an integer type and also part of a sequence correct?

Reply:
I saw some example, id start with p for proteomics that why i choose text as datatype but we could discuss about it , when you have time